### PR TITLE
Add 0xc000007f to mapping

### PR DESCRIPTION
**Pull request by rsdoherty**
Adds the low disk space code to the mapping, so a more descriptive error is raised.

From https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-cifs/8f11e0f3-d545-46cc-97e6-f00569e3e1bc:

image

### CODE REVIEW

**Review by jborean93 — COMMENTED** [review]
The Microsoft NTSTATUS page lists this error code as `STATUS_DISK_FULL` https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/596a1078-e883-4972-9bbc-49e60bebca55. I would probably prefer to keep these values aligned if possible sorry.

**Comment by rsdoherty** [comment]
Makes sense - thanks!

File: src/smbprotocol/exceptions.py [hunk 1]
```diff
@@ -612,6 +612,11 @@ class ServerUnavailable(SMBResponseException):
     _STATUS_CODE = NtStatus.STATUS_SERVER_UNAVAILABLE
 
 
+class NotEnoughSpaceOnTheDisk(SMBResponseException):
```
Comment (jborean93) [inline src/smbprotocol/exceptions.py:615]: Sorry to be a pain but can we make this `DiskFull` to reflect the status code?

**Review by jborean93 — APPROVED** [review]

**Comment by jborean93** [comment]
Thanks for adding this and appreciate you taking the time to make the changes after my review!

**Comment by rsdoherty** [comment]
No problem! I'm actually contributing this as I'm using pypsexec for some automation, so I'd be keen to get this into a release that's on pypi so I can bump the dependency in my application if possible, please?

**Comment by rsdoherty** [comment]
Hey @jborean93 - are you able to review the above please?

**Comment by jborean93** [comment]
Sorry I will look at pushing a new release tomorrow.

**Comment by jborean93** [comment]
v1.12.0 has been pushed to PyPI with these changes https://pypi.org/project/smbprotocol/.

**Comment by rsdoherty** [comment]
Thanks!

**Pull request merged**